packages/depot_tools.rb

```diff
@@ -23,26 +23,24 @@ def self.install
       FileUtils.rm_rf 'man/src'
       FileUtils.rm_rf Dir.glob('.git*')
       system 'find -name \'*.bat\' -exec rm {} +'
-      FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/depot_tools"
-      FileUtils.mv 'man/', "#{CREW_DEST_PREFIX}/share"
-      FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/share/depot_tools"
+      FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/libexec/depot_tools"
+      FileUtils.mv 'man/', "#{CREW_DEST_PREFIX}/libexec"
```

Comment:
Why would we want to move a man directory here?

Reply:
We wouldn't, it's fixed.